crates/bevy_app/src/propagate.rs

```diff
@@ -33,15 +34,31 @@ use bevy_reflect::Reflect;
 /// Individual entities can be skipped or terminate the propagation with the [`PropagateOver`]
 /// and [`PropagateStop`] components.
 ///
-/// Propagation occurs during [`Update`] in the [`PropagateSet<C>`] system set.
+/// By default the propagation occurs during [`Update`] schedule in the [`PropagateSet<C>`] system set.
```

Comment:
IIRC we wanted to change this to `PostUpdate`. Can we do that in this PR too?

Reply:
#20356 is the issue for that.

Reply:
Does it need a default? It might be clearer to always require users to specify a ScheduleLabel like we we do with `add_system` etc.

Reply:
Done!

Reply:
I was going to suggest removing the default as well, but I was too tired by the time I posted this PR.
Glad it worked out! :)